app/dbOperations/subjects.go

```diff
@@ -49,11 +50,11 @@ func GetSubjectByName(cosmos gremcos.Cosmos, name string) (model.Subject, error)
 	return getSubjectFromResponse(res)
 }
 
-func CreateSubject(cosmos gremcos.Cosmos, subject model.Subject) (model.Subject, error) {
+func CreateSubject(cosmos gremcos.Cosmos, subject apiModel.Subject) (storageModel.Subject, error) {
```

Comment:
In fact it should receive an storageModel instead of apiModel (same as the return), since it is just the data base part. The conversion should be in the handler (I don't know if we will need an intermediate thing to process these conversions)

Reply:
Done!